### PR TITLE
Bluetooth page design

### PR DESCRIPTION
**Pull request by nn111x**
please review

### CODE REVIEW

File: Front-End/public/connect-esp-intro.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    color: #333;
+    background-color: #f9f9f9;
+}
+
+main {
+    max-width: 800px;
+    margin: 20px auto;
+    background: #fff;
+    padding: 20px;
+    border: 1px solid #ddd;
```
Comment (DimitarSamarov07) [inline Front-End/public/connect-esp-intro.css]: Unnecessary. Please remove this line. The border can't be seen anyway.

File: Front-End/public/connect-esp-intro.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    margin-bottom: 10px;
+}
+
+.instructions {
+    margin-bottom: 20px;
+}
+
+.instructions p {
+    margin: 10px 0;
+    line-height: 1.6;
```
Comment (DimitarSamarov07) [inline Front-End/public/connect-esp-intro.css]: Line height not need, please remove

**Review by DimitarSamarov07 — CHANGES_REQUESTED** [review]
Please read the review and make the changes.

File: Front-End/public/connect-esp-intro.css (hunk 1 above)
Comment (nn111x) [reply Front-End/public/connect-esp-intro.css]: This was done so the box can look better. There's no need to remove it.

Comment (DimitarSamarov07) [reply Front-End/public/connect-esp-intro.css]: I see, disregard it.

**Review by DimitarSamarov07 — APPROVED** [review]
Looks good now.

**Pull request merged**